packages/tokens/styles/type.scss

```diff
@@ -0,0 +1,29 @@
+@import "color";
+@import "layers";
+
+$ca-grid: 1.5rem; // 24px @ default root font-size of 16px
+
+$ca-weight-extra-light: 200;
+$ca-weight-light: 300;
+$ca-weight-book: 400;
+$ca-weight-medium: 500;
+$ca-weight-semibold: 500; // Note: in Sketch, semibold is 600. But murmur has an existing value of semibold=500 that is heavily used.
+
+// Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
+$ca-default-font-family: "Open Sans", Helvetica, Arial, sans-serif;
+$ca-default-font-base-size: 0.875rem; /* 14px */
+$ca-default-font-descender-height: 0.115;
+
+// Combine these into a Sass map ($ca-default-font) once node-sass includes libsass 3.5.0.beta.3 with this bug fix: https://github.com/sass/libsass/issues/2309
+$ca-ideal-sans-font-family: "Ideal Sans A", "Ideal Sans B",
+  $ca-default-font-family;
+$ca-ideal-sans-font-base-size: 1rem; /* 16px */
+$ca-ideal-sans-font-descender-height: 0.14;
+
+// Locale-specific fonts
```

Comment:
Locale-specific fonts can probably go too, WDYT?

Reply:
Hmmm, we do want these to apply in every repo though. I see them as the same as the standard type styles but for other languages. They need more fleshing out though.